registry/server/routes/index.ts

```diff
@@ -6,3 +6,4 @@ export { default as sharedProps } from '../sharedProps/routes';
 export { default as authEntities } from '../authEntities/routes';
 export { default as settings } from '../settings/routes';
 export { default as versioning } from '../versioning/routes';
+export { default as routerDomains } from '../routerDomains/routes';
```

Comment:
I think you also need to add filtration capability by domainId to getRoutes API

Reply:
sorry, can't understand, what do you mean?

Reply:
I want to be able to filter routes list by `domainId`, in routes API

Reply:
done